src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,136 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage{
+
+    public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+
+    @FindBy(xpath = "//img[@class='avatar']")
+    public WebElement homeIcon;
+    @FindBy(xpath = "//img[@class='avatar']")
+    public WebElement userProfileIcon;
+    @FindBy(xpath = "//a[@class='logout control']")
+    public WebElement logoutButton;
+    @FindBy(css = "i[class='fa fa-plus-circle create']")
+    private WebElement plusIcon;
+    @FindBy(css = "li[data-testid='playlist-context-menu-create-smart']")
+    private WebElement newSmartPlaylistButton;
+    @FindBy(css = "input[name='name']")
+    private WebElement playlistNameField;
+    @FindBy(xpath = "//input[@name='name']")
+    public WebElement smartPlaylistNameField;
+    @FindBy(xpath = "//div[@data-test='smart-playlist-rule-row'][1]//select[@name='model[]']/option[1]")
+    public WebElement RuleOneTitle;
+    @FindBy(xpath = "//div[@data-test='smart-playlist-rule-row'][1]//select[@name='operator[]']/option[3]")
+    public WebElement RuleOneContains;
+    @FindBy(xpath = "//div[@data-test='smart-playlist-rule-row'][1]//input[@type='text']")
+    public WebElement RuleOneTextField;
+    @FindBy(xpath = "//div[@class='smart-playlist-form']//button[@type='submit']")
+    public WebElement saveSmartPlaylistBtn;
+    @FindBy(css = "div.alertify-logs.top.right")
+    private WebElement playlistCreatedMessageLocator;
+    @FindBy(css = "div.success.show")
+    public WebElement successNotification;
+
+    //Page Elements
+    By userAvatarIcon = By.cssSelector("img.avatar");
+
+    By allSongsList = By.cssSelector("li a.songs");
+    By byPlaylistCreatedMessageLocator= By.cssSelector("div.alertify-logs.top.right");
+    By logoutBtn = By.xpath("//a[@class='logout control']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```